src/Microsoft.AspNetCore.Sockets.Client/LongPollingTransport.cs

```diff
@@ -39,104 +41,101 @@ public void Dispose()
         {
             _senderCts.Cancel();
             _pollCts.Cancel();
-            _pipeline?.Dispose();
         }
 
-        public Task StartAsync(Uri url, IPipelineConnection pipeline)
+        public Task StartAsync(Uri url, IChannelConnection<Message> toFromConnection)
```

Comment:
This naming is throwing me off. Can we name things similar to the server? Transport and Application?